src/db/cass/create_db.py

```diff
@@ -21,3 +21,6 @@
 
 # Users table for querying with username
 cluster_services.create_table_schema('CREATE TABLE IF NOT EXISTS users_name (user_id text, username text, password text, sid text, PRIMARY KEY(username))')
+
+# Tables for storing OpenCV frames
+cluster_services.create_table_schema('CREATE TABLE IF NOT EXISTS cv_frames (camera_id text, stream_id text, frame_id text, frame blob, objects_detected blob, PRIMARY KEY(stream_id, camera_id, frame_id))')
```

Comment:
If you want to sort the frames, I wouldn't recommend using `camera_id` as the clustering key. I would change `frame_id` to an `int` type and use that as the clustering key. So I would model my PRIMARY KEYS like this:

If you want sorting:  Change `frame_id` type to int and use `PARIMARY KEY((camera_id, stream_id), frame_id)`. Then I would do SELECT queries using the `camera_id` and `stream_id` as lookup and the data returned will automatically be in sorted order based on the `frame_id`.

If you don't want sorting: `PRIMARY KEY((camera_id, stream_id))` and use both of those in your WHERE clause when looking up frames.

But I still think you should split up the tables into `frame_metadata` and `frame_data` that I did. hehe